oc_anonymizer.py

```diff
@@ -17,47 +17,59 @@ class PlistStripper:
-        self.censor_misc_boot()
-        self.censor_misc_debug()
-        self.censor_misc_security()
-        self.censor_platforminfo_generic()
-        self.censor_uefi_apfs()
+        self.reset_misc_boot()
+        self.delete_misc_blessoverride()
+        self.reset_misc_debug()
+        self.delete_misc_entries()
+        self.reset_misc_security()
+        self.delete_platforminfo_generic()
+        self.disable_uefi_apfs()
         self.dump()
 
-    def censor_misc_boot(self) -> None:
+    def reset_misc_boot(self) -> None:
         """ Sets Misc/Boot/LauncherOption to Disabled to avoid registering the launcher option in the firmware
         preferences for persistence"""
 
         self.plist['Misc']['Boot']['LauncherOption'] = 'Disabled'
 
-    def censor_misc_debug(self) -> None:
+    def delete_misc_blessoverride(self) -> None:
+        """ Deletes custom BlessOverride entries from config.plist"""
+
+        self.plist['Misc']['BlessOverride'] = []
+
+    def reset_misc_debug(self) -> None:
         """ Sets Misc/Debug/Target to 3"""
 
         self.plist['Misc']['Debug']['Target'] = 3
 
-    def censor_misc_security(self) -> None:
+    def delete_misc_entries(self) -> None:
+        """ Deletes custom bootloader entries from config.plist"""
+
+        self.plist['Misc']['Entries'] = []
+
+    def reset_misc_security(self) -> None:
         """Sets
             -> Misc/Security/ApECID to 0 to disallow using personalised Apple Secure Boot identifiers (unsupported from
                 Monterey and later)
             -> Misc/Security/ScanPolicy = 0 to allow OpenCore's operating system detection policy
-            -> Misc/Security/SecureBootModel = Default to configure Apple Secure Boot hardware model
+            -> Misc/Security/SecureBootModel = Disabled. Disables Apple Secure Boot hardware model to avoid issues during Installation. Re-enable in Post-Install so System Updates work when using an SMBIOS of a Mac model with a T2 Security Chip.
             -> Misc/Security/Vault = Optional to disable OpenCore's vaulting mechanism
         """
 
         self.plist['Misc']['Security']['ApECID'] = 0
         self.plist['Misc']['Security']['ScanPolicy'] = 0
-        self.plist['Misc']['Security']['SecureBootModel'] = 'Default'
+        self.plist['Misc']['Security']['SecureBootModel'] = 'Disabled'
         self.plist['Misc']['Security']['Vault'] = 'Optional'
 
-    def censor_platforminfo_generic(self) -> None:
+    def delete_platforminfo_generic(self) -> None:
         """Censors several SMBIOS identification fields"""
 
         self.plist['PlatformInfo']['Generic']['MLB'] = 'XX-CHANGE_ME-XX'
         self.plist['PlatformInfo']['Generic']['ROM'] = b'\x11"3DUf'
         self.plist['PlatformInfo']['Generic']['SystemSerialNumber'] = 'XX-CHANGE_ME-XX'
         self.plist['PlatformInfo']['Generic']['SystemUUID'] = 'XX-CHANGE_ME-XX'
 
-    def censor_uefi_apfs(self) -> None:
+    def disable_uefi_apfs(self) -> None:
```

Comment:
I'd rename it to `fix_uefi_apfs` as it sets both `MinVersion` and `MinDate` to `-1` to allow scanning every APFS like partition. What are your thoughts on it?

Reply:
Good idea.